server/fig.yml

```diff
@@ -1,41 +0,0 @@
-db:
```

Comment:
don't we need fig anymore? there is still used in README.md
please remove it from there also if it is the case.